app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not command.startswith("mv"):
+        print("Error: invalid command")
+        return
+
+    src, dst = command.split()[1:]
```

Comment:
Your code will not work with this
command = "mv file1.txt"
command = "mv file1.txt file2.txt file3.txt"
It will fails with unknown error